lib/Test.pm6

```diff
@@ -39,6 +39,8 @@ my int $done_testing_has_been_run = 0;
 _init_vars();
 
 sub _init_io {
+    nqp::setbuffersizefh(nqp::getstdin(),  0);
```

Comment:
Setting it on STDIN makes no sense; the op is only about output buffering. Setting it on both STDOUT and STDERR might make sense.

Reply:
do'h! That's exactly what I meant…